pyrgg/params.py

```diff
@@ -10,8 +10,8 @@
         1- Pyrgg engine
         """
         )],
-    2: ["file_name", "- File Name : "],
-    3: ["number_of_files", "- Number of Files : "],
+    2: ["file_name", "- File Name (not empty) : "],
+    3: ["number_of_files", "- Number of Files (>=0) : "],
```

Comment:
`>=0` or `>=1` ?

Reply:
I think it makes more sense it it's `>=0`. Image the case that you just wanted to make configs.